packages/data/src/constants.ts

```diff
@@ -10,3 +10,5 @@ export const DEFAULT_CLUE_RANKS: readonly number[] = [1, 2, 3, 4, 5];
 
 export const MIN_PLAYERS = 2;
 export const MAX_PLAYERS = 6;
+
+export const MAX_CARDS_IN_A_DECK = 60;
```

Comment:
should probably be a composition of MAX_CARDS_IN_A_SUIT and MAX_SUITS_IN_A_VARIANT